nin/dasBoot/Loader.js

```diff
@@ -89,8 +93,12 @@ Loader.prototype.start = function(onprogress, oncomplete) {
       var response = null;
       var request = new XMLHttpRequest();
       request.open('GET', Loader.rootPath + item.filepath, 1);
+      if(item.options.responseType) {
+        request.responseType = item.options.responseType;
+      }
       request.onload = function() {
-        item.callback(request.responseText);
+        item.callback(item.options.responseType == 'arraybuffer' ? request.response
```

Comment:
I get this error in Chrome/Opera, but not in FireFox:
Uncaught TypeError: item.callback is not a function